docs/pages/machine-id/workload-identity.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: Workload Identity
+description: Teleport's Workload Identity for SPIFFE
+---
+
+<Admonition type="tip" title="Preview">
+Teleport Workload Identity is currently in Preview. We are actively working on
+improving the feature and would love to [hear your feedback](mailto: product@goteleport.com).
```

Comment:
The Markdown syntax renders on the dev server. Maybe remove the space after the colon?

Reply:
2285838b13